README.md

```diff
@@ -0,0 +1,49 @@
+# Integreat Compass
+
+This system gathers courses that job center employees can pass on in counseling sessions to refugees and newcomers.
+
+## TL;DR
+
+### Prerequisites
+
+Following packages are required before installing the project (install them with your package manager):
+
+* `npm` version 7 or later
+* `nodejs` version 12 or later
+* `python3` version 3.9 or later
+* `python3-pip` (Debian-based distributions) / `python-pip` (Arch-based distributions)
+* `python3-venv` (only on Debian-based distributions)
+* Either `postgresql` **or** `docker` to run a local database server
+
+### Installation
+
+````
+git clone git@github.com:digitalfabrik/integreat-compass.git
+cd integreat-compass
+./tools/install.sh
+````
+
+### Run development server
+
+````
+./tools/run.sh
+````
+
+* Go to your browser and open the URL `http://localhost:8000`
```

Comment:
I'd prefer if it would be possible to run integreat-compass alongside integreat-cms and lunes-cms if we need to quickly switch between projects in development...
So maybe a different port than `8000` and `8080`? Not sure what other ports are in use by other app projects...